be/src/Unic.UrlMapper2/code/Services/RedirectionService.cs

```diff
@@ -102,7 +102,7 @@ protected virtual Redirect GetRegexMatch(RedirectSearchData redirectSearchData,
             if (string.IsNullOrWhiteSpace(sourceTerm)) return default;
 
             // We are going to take the one regex redirect which has the longest match within the term
-            var regexMatches = enumerableRedirects.Where(r => r.RegexEnabled && Regex.IsMatch(input: sourceTerm, pattern: r.Term));
+            var regexMatches = enumerableRedirects.Where(r => (r.RegexEnabled) && Regex.IsMatch(input: sourceTerm, pattern: r.Term));
```

Comment:
Are the braces necessary?

Reply:
Not necessary :) Probably it's leftover after my testing or maybe code cleanup change. Don't remember now